packages/editor/src/client/blocks/teaserGridBlock.tsx

```diff
@@ -332,12 +338,12 @@ export function TeaserContent({
         <Box display="flex" flexWrap="wrap">
           <Box flexShrink={0} marginRight={Spacing.ExtraSmall}>
             <Typography variant="subtitle1" color="gray">
-              Style: {labelForTeaserStyle(style)}
+              {t('Style')}: {labelForTeaserStyle(style)}
```

Comment:
You should use interpolation for those translations. https://www.i18next.com/translation-function/interpolation

Reply:
Done